File: web-ui/src/displays/Page.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractPage from "../../gen/displays/AbstractPage";
import PageNotifier from "../../gen/displays/notifiers/PageNotifier";
import PageRequester from "../../gen/displays/requesters/PageRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';

const styles = theme => ({});

class Page extends AbstractPage {

	constructor(props) {
		super(props);
		this.notifier = new PageNotifier(this);
		this.requester = new PageRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(Page);
DisplayFactory.register("Page", withStyles(styles, { withTheme: true })(Page));

File: framework/ui/src/io/intino/alexandria/ui/displays/Page.java
package io.intino.alexandria.ui.displays;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.PageNotifier;

public class Page<DN extends PageNotifier, B extends Box> extends Display<DN, B> {

	public Page(B box) {
		super(box);
	}

}